docs/pages/inboxes/manage-inboxes.mdx

```diff
@@ -0,0 +1,189 @@
+# Manage XMTP inboxes
+
+The first time someone uses an installation of your app, your app creates an inbox ID and installation ID associated with their wallet address. To do this, you [create a client](https://docs.xmtp.org/inboxes/build-inbox#create-an-xmtp-client) for their wallet address. 
```

Comment:
> The first time someone uses an installation of your app, your app creates an inbox ID and installation ID associated with their wallet address.

They may or may not be creating an inbox ID. If you've already used another XMTP app with the same wallet, you'll use the already existing inbox ID by default